src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -577,7 +558,7 @@ std::shared_ptr<GLHEVertProps> GetVertProps(EnergyPlusData &state, std::string c
 std::shared_ptr<GLHEVertSingle> GetSingleBH(EnergyPlusData &state, std::string const &objectName)
 {
     // Check if this instance of this model has already been retrieved
-    for (auto &thisBH : state.dataGroundHeatExchanger->singleBoreholesVector) {
+    for (auto const &thisBH : state.dataGroundHeatExchanger->singleBoreholesVector) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:580]:(style),[constVariable],Variable 'thisBH' can be declared as reference to const